charts/jaeger/values.yaml

```diff
@@ -0,0 +1,15 @@
+# Default values for tracing.
+
+namespace: linkerd-jaeger
+
+collector:
+  image: 
+    name: omnition/opencensus-collector
+    version: 0.1.11
+    pullPolicy: Always
```

Comment:
do we actually want the default pullPolicy to be Always for these published images?